### PR TITLE
Add plain text variant for email

### LINKED ISSUE #187: Plain text variant for better spam score
| Q | A
| --- | ---
| Bug? | no
| New Feature? | yes
| Bundle Version | 1.0.0-RC5 
| Sulu Version | 1.6.26
| Browser Version | Browser name and version

#### Actual Behavior

Now only an html variant is send to the customer. Mails with no text/plain variant get lower scores. See https://litmus.com/blog/best-practices-for-plain-text-emails-a-look-at-why-theyre-important.

#### Expected Behavior

Also a text/plain variant is send

#### Steps to Reproduce

Send mail via formbundle an validate via mailtester.com

#### Possible Solutions

I will create a Pull Request with simple text/plain variant.

### PR DESCRIPTION
**Pull request by kleinkoerkamp**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #187 
| License | MIT

#### What's in this PR?

Default plain text variant for better spam score, see #187. It's overridable

#### To Do

- [ ] Update CHANGELOG.md
- [ ] Create documentation

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@kleinkoerkamp thank you for your PR and sorry for the late response will try to have a look at this at the weekend!

**Comment by alexander-schranz** [comment]
@kleinkoerkamp looks good for me can you rebase from master branch? 
Add the new configuration to the docs: https://github.com/sulu/SuluFormBundle/blob/master/Resources/doc/dynamic.md#custom-emails
And add a CHANGELOG.md entry

File: Configuration/MailConfiguration.php [hunk 1]
```diff
@@ -260,6 +265,28 @@ public function setTemplate($template)
...
+    }
+
+    /**
+     * Set plain text template.
+     *
+     * @param string $plainTextTemplate
+     *
+     * @return $this
+     */
+    public function setPlainTextTemplate(string $plainTextTemplate)
```
Comment (alexander-schranz) [inline Configuration/MailConfiguration.php]: @kleinkoerkamp `string` is not supported in php5.5

**Comment by alexander-schranz** [comment]
@kleinkoerkamp Thank you for this feature!

**Comment by kleinkoerkamp** [comment]
@alexander-schranz no problem. Any idea when this will be released?

**Comment by alexander-schranz** [comment]
@kleinkoerkamp released in 1.0.0-RC7

**Pull request merged**